pkg/kubewarden/chart/kubewarden/policy-server/index.vue

```diff
@@ -41,7 +43,47 @@ export default {
   },
 
   data() {
-    return { chartValues: this.value };
+    if (!this.value.spec?.securityContexts) {
+      this.value.spec.securityContexts = {};
+    }
+
+    if (!this.value.spec?.securityContexts?.container) {
+      this.value.spec.securityContexts.container = {};
+    }
+
+    if (!this.value.spec?.securityContexts?.pod) {
+      this.value.spec.securityContexts.pod = {};
+    }
+
+    // defaults for this.value.spec.securityContexts.container object properties
+    [
+      ['capabilities', {}],
+      ['seLinuxOptions', {}],
+      ['seccompProfile', {}],
+      ['windowsOptions', {}],
+    ].forEach((item) => {
+      if (!this.value.spec?.securityContexts?.container[item[0]]) {
+        this.value.spec.securityContexts.container[item[0]] = item[1];
+      }
+    });
```

Comment:
Nit: This could be simplified to an array of strings since all of their default values are objects, but I assume you're keeping it consistent with the next stanza.

Reply:
Exactly